app/pb/api.proto

```diff
@@ -219,6 +234,13 @@ message RepeatedStringValue {
   repeated string value = 2;
 }
 
+message RepeatedLanguageAbilityValue {
+  // A RepeatedLanguageAbilityValue is like a "protobuf wrapper" (like {Type}Value) but for repeated LanguageAbility
+  // allows us to determine if the user wanted to remote this list of LanguageAbility's
+  bool exists = 1;
```

Comment:
Do we need `exists`? A RepeatedLanguageAbilityValue could be absent in a rpc call without this extra field.

Reply:
good question, @aapeliv ?

Reply:
Note that RepeatedStringValue also has exists

Reply:
It seems it's not needed, as a non-existent submessage is not sent while an empty but existing submessage is serialized and so can be detected. It'd be great to do a PR to fix all of them and document how to use this stuff and when one can detect the existence/absence of fields. Otherwise I'd be in favor of leaving it in for clarity (in this PR as well).

Reply:
Yeah, we can remove `exists` from RepeatedStringValue as well. Sorry for a sloppy review [that time](https://github.com/Couchers-org/couchers/commit/3931d6955a7d805f048ff0c4b6b1c9498ce9045a).

Reply:
Sorry but I insist in removing `exists` for `RepeatedLanguageAbilityValue` in this PR.

Reply:
OK, I'll do that